### PR TITLE
Feature/coverage ci python

### PR DESCRIPTION
**Pull request by leunMar**
Add python package to CI coverage jobs

on top of #219

### CODE REVIEW

File: .coverage.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    ici_exec_in_workspace "$extend" "$ws" catkin build $pkg -v --no-deps --catkin-make-args ${pkg}_coverage
+    cd $TARGET_REPO_PATH
+
+    if [ -a $ws/build/$pkg/${pkg}_coverage.info.cleaned ]; then
+        echo "Coverage summary for $pkg ----------------------"
+        lcov --summary $ws/build/$pkg/${pkg}_coverage.info.cleaned
+        echo "---------------------------------------------------"
+
+        line_cov_percentage=$(lcov --summary $ws/build/$pkg/${pkg}_coverage.info.cleaned 2>&1 | grep -Poi "lines\.*: \K[0-9.]*")
+    else
```
Comment (HiddenBug) [inline .coverage.sh:22]: I admittedly have not checked in real life what you do here. But just humor me:  What happens if I have a package without C++ or Python code or more generally an empty package with empty src folder? Do I get an error or something, even though I have no code to check?

**Review by HiddenBug — CHANGES_REQUESTED** [review]
A question.

File: .coverage.sh (hunk 1 above)
Comment (leunMar) [reply .coverage.sh:22]: You will get an error if it is included here:

>       DOCKER_RUN_OPTS='-e COVERAGE_PKGS="pilz_extensions pilz_robot_programming"'

Comment (HiddenBug) [reply .coverage.sh:22]: Is this the behavior we want?

Comment (leunMar) [reply .coverage.sh:22]: Yes. If you have no code to check, you should not include the package like above.

**Review by HiddenBug — APPROVED** [review]
Looks good to me. I assume you have checked locally that it fails if the coverage in python is not 100%?

**Comment by leunMar** [comment]
> Looks good to me. I assume you have checked locally that it fails if the coverage in python is not 100%?

Thanks, but something is still wrong. Since I re-added pilz_trajectory_generation to the package list it does no longer work...

Yes, I did check it locally.

**Pull request merged**